src/cmds/restic/integration_test.go

```diff
@@ -516,23 +516,23 @@ func TestBackupExclude(t *testing.T) {
 		testRunBackup(t, []string{datadir}, opts, gopts)
 		snapshots, snapshotID := lastSnapshot(snapshots, loadSnapshotMap(t, gopts))
 		files := testRunLs(t, gopts, snapshotID)
-		Assert(t, includes(files, filepath.Join("testdata", "foo.tar.gz")),
+		Assert(t, includes(files, filepath.Join("/testdata", "foo.tar.gz")),
```

Comment:
Might be more conforming, yes. Though weirdly, the one OS I expected to have issues here, succeeded with the test. I'll make the change.